hack/update-schemas.sh

```diff
@@ -18,20 +18,34 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# Create a backup for every linked CRD.
-links=$(find "$(dirname "$0")/../config/core/300-resources" -type l)
-for link in $links; do
-  cp "$link" "$link.bkp"
-done
-
-# Make sure you install the patched version of controller-gen from
-# https://github.com/markusthoemmes/controller-tools/tree/knative-specific
+# Install patched schemagen into a temporary directory.
 #
 # We need a patched version because
 # 1. There's a bug that makes our URL types unusable
 #    see https://github.com/kubernetes-sigs/controller-tools/issues/560
 # 2. We need specialized logic to filter down the surface of PodSpec we allow in Knative.
 #    The respective config for this is in `schemapatch-config.yaml`
+export GOBIN
+GOBIN=$(mktemp -d)
```

Comment:
Do we need to clean this up or does that happen for free?

Reply:
For free AFAIK